src/main/java/com/algorand/algosdk/v2/client/algod/AccountInformation.java

```diff
@@ -0,0 +1,40 @@
+package com.algorand.algosdk.v2.client.algod;
+
+import com.algorand.algosdk.v2.client.common.Client;
+import com.algorand.algosdk.v2.client.common.Query;
+import com.algorand.algosdk.v2.client.common.QueryData;
+import com.algorand.algosdk.v2.client.common.Response;
+import com.algorand.algosdk.v2.client.model.Account;
+
+
+/**
+ * Given a specific account public key, this call returns the accounts status, 
+ * balance and spendable amounts /v2/accounts/{address} 
+ */
+public class AccountInformation extends Query {
+
+	private String address;
+
+	/**
+	 * @param address An account public key 
+	 */
+	public AccountInformation(Client client, String address) {
```

Comment:
This can be an `Address` object, perhaps overloaded so that there are 2 constructors:
`public AccountInformation(Client client, String address) {`
and
`public AccountInformation(Client client, Address address) {`

Reply:
fixed. 
I am not in favor of overloading these methods.
Address constructor is too simple already.